### DISCUSSION TITLE
Assessing Green Software Maturity in Projects

### DISCUSSION THREAD

**Discussion by gabdss** [Green-Software-Foundation/discussions#43] [Ask Anything]
Hi everyone,

I'm part of a team that is getting into sustainable software development (I'm a sustainability expert, not a coder). To drive our efforts, I want to evaluate our software projects and gauge our current level of green software maturity. This will help us set goals and enhance our green coding approach.

I'd appreciate your assistance in the following:

1. How can we initiate the assessment of our software projects to understand our current level of green software maturity?
2. What criteria and metrics do you recommend for this assessment?
3. Based on your experience, what practices or initiatives are typically associated with each maturity level (initial to expert)?

Any articles, ideas, or insights you can share would be highly valuable.

Thank you!

**Comment by ArneTR** [comment]
Hey @gabdss,

it reads like you have already done some actions and now are looking for "what else it out there" or how you compare to something like an official roadmap or some other companies.

Do you mind sharing what measures you have already implemented? Would be curious.
